sklearn/calibration.py

```diff
@@ -139,18 +139,25 @@ class CalibratedClassifierCV(BaseEstimator, ClassifierMixin,
 
     References
     ----------
-    .. [1] Obtaining calibrated probability estimates from decision trees
-           and naive Bayesian classifiers, B. Zadrozny & C. Elkan, ICML 2001
-
-    .. [2] Transforming Classifier Scores into Accurate Multiclass
-           Probability Estimates, B. Zadrozny & C. Elkan, (KDD 2002)
-
-    .. [3] Probabilistic Outputs for Support Vector Machines and Comparisons to
-           Regularized Likelihood Methods, J. Platt, (1999)
-
-    .. [4] Predicting Good Probabilities with Supervised Learning,
+    .. [1] `Obtaining calibrated probability estimates from decision trees
```

Comment:
I think we need to discuss if we want references to papers in the API docs. I've seen opinions on both sides of this.

Reply:
No worries, I'll close this for now. Thanks!